tools/test

```diff
@@ -117,8 +134,32 @@ run_native_android() {
     )
 }
 
+run_native_ios() {
+    :
+    # TODO: At least make an iOS build when $platform is `ios` or
+    # `both`.
+}
+
 run_native() {
-    run_native_android || return
+    if [[ $platform == android || $platform == both || $platform == sloppy ]]; then
+        echo 'Running Android native tests...';
```

Comment:
> Maybe I'll try to experiment with something later.

Cool, that'd be helpful, thanks! I'm also happy to act on some sketched-out ideas, but it might be more efficient for you to implement what you have in mind.